Add meta descriptions to Lisk-SDK, closes #371 (#387)
LiskHQ/lisk-docs: modules/ROOT/pages/lisk-elements/packages/api-client/signatures.adoc

= Lisk Elements API Client: Signatures Resource
Mona Bärenfänger <mona@lightcurve.io>

> :description: Technical reference for the Signatures endpoints of the API Client package of Lisk Elements. It copmrises Usage examples, available parameters and example responses.

Reviewer: ```suggestion
:description: Technical references regarding the Signatures endpoints of the API Client package of Lisk Elements.This consists of usage examples, available parameters and example responses.
```

> :description: Technical references regarding the Signatures endpoints of the API Client package of Lisk Elements.This consists of usage examples, available parameters and example responses.

Reviewer: ```suggestion
:description: Technical references regarding the Signature endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

Author: I would use plural here, because the endpoint is in plural as well

Reviewer: Agreed.
